### PR TITLE
Remove log4j banner from homepage

### PR DESCRIPTION
**Pull request by abeatrix**
Remove log4j banner from dot com as requested in https://github.com/sourcegraph/sourcegraph/issues/29387 
image
image

Note: I haven't deleted the `VulnerabilityAnnouncement` Component for future use but I can delete it if that's not the best practice, just let me know. Thanks!

### CODE REVIEW

**Review by rrhyne — APPROVED** [review]
lgtm

File: client/web/src/search/home/VulnerabilityAnnouncement.tsx [hunk 1]
```diff
@@ -4,46 +4,42 @@ import React from 'react'
```
Comment (erzhtor) [inline client/web/src/search/home/VulnerabilityAnnouncement.tsx:4]: Should we remove this component/CSS file as well?

**Review by erzhtor — APPROVED** [review]
LGTM 👍

File: client/web/src/search/home/VulnerabilityAnnouncement.tsx (hunk 1 above)
Comment (abeatrix) [reply client/web/src/search/home/VulnerabilityAnnouncement.tsx:4]: I was wondering if we can keep it for future use but I guess not? I will remove this component as well as the CSS file!

**Pull request merged**